jsonschema python lib missing in latest universal container image

### Issue Summary

When using the dev branch in the latest universal container I get the following error when trying to upload to CVP using ansible-cvp collection:

```
TASK [arista.avd.eos_config_deploy_cvp : Create configlets on CVP cvaas.] ******
[31](https://github.com/bjmeuer/L3LS_Automation/actions/runs/11250881751/job/31281346023#step:8:32)fatal: [cvaas]: FAILED! => {"changed": false, "msg": "JSONSCHEMA is required. Please install using pip install jsonschema"}
```


### Which component(s) of AVD impacted

eos_config_deploy_cvp

### How do you run AVD ?

None

### Steps to reproduce

```shell
Playbook:

---
#
# This playbook builds configs and deploys them using Ansible CVP
# use build or deploy tags to filter actions if required
#
- name: Configuration upload for Fabric
  hosts: cvaas
  connection: local
  gather_facts: false
  collections:
    - arista.cvp
  tasks:
    - name: "Deploying Fabric configurations to CVaaS"
      tags: [deploy]
      import_role:
        name: arista.avd.eos_config_deploy_cvp
      vars:
        container_root: DC_ALL_TEST
        configlets_prefix: Config
        state: present
        cv_collection: v3
        execute_tasks: true



Relevant Jobs from the CI pipeline

---
name: Run L3LS CI Pipeline

on:
  push:
    branches: [ "main" ]
jobs:

  build-job:
    runs-on: self-hosted
    container: 
      image: ghcr.io/aristanetworks/avd/universal:latest
      env:
        PYTHONPATH: "/home/avd/.local/lib/python3.11/site-packages"
    needs: [run-lint]
    steps:
      - run: sudo chown -R avd:avd $GITHUB_WORKSPACE
      - run: chmod -R 755 $GITHUB_WORKSPACE
      - run: sudo chown -R avd:avd /github
      - name: Checkout code
        uses: actions/checkout@v4
      - run: ansible-playbook -i ./l3ls_automation/inventory_test.yml playbooks/build-configs_test.yml
      - name: Upload pyavd result
        uses: actions/upload-artifact@v4
        with:
          name: l3ls_automation
          path: /__w/L3LS_Automation/L3LS_Automation
          retention-days: 7

  deploy-job:
    runs-on: self-hosted
    container: 
      image: ghcr.io/aristanetworks/avd/universal:latest
      env:
        PYTHONPATH: "/home/avd/.local/lib/python3.11/site-packages"
    needs: [build-job]
    steps:
      - run: sudo chown -R avd:avd $GITHUB_WORKSPACE
      - run: chmod -R 755 $GITHUB_WORKSPACE
      - run: sudo chown -R avd:avd /github
      - name: Download pyavd result
        uses: actions/download-artifact@v4
        with:
          name: l3ls_automation
          path: /__w/L3LS_Automation/L3LS_Automation
      - run: ansible-playbook -i ./l3ls_automation/inventory_test.yml playbooks/fabric-deploy_test.yml
```


### Relevant log output

```shell
Logs from the deploy job from the CI pipeline:


2024-10-09T08:24:12.1401837Z Current runner version: '2.319.1'
2024-10-09T08:24:12.1548557Z Runner name: 'containerlab2'
2024-10-09T08:24:12.1549420Z Runner group name: 'Default'
2024-10-09T08:24:12.1550566Z Machine name: 'containerlab2'
2024-10-09T08:24:12.1579005Z Testing runner upgrade compatibility
2024-10-09T08:24:12.4032162Z ##[group]GITHUB_TOKEN Permissions
2024-10-09T08:24:12.4034974Z Contents: read
2024-10-09T08:24:12.4035487Z Metadata: read
2024-10-09T08:24:12.4035953Z Packages: read
2024-10-09T08:24:12.4036401Z ##[endgroup]
2024-10-09T08:24:12.4041446Z Secret source: Actions
2024-10-09T08:24:12.4042545Z Prepare workflow directory
2024-10-09T08:24:12.6058538Z Prepare all required actions
2024-10-09T08:24:12.6370263Z Getting action download info
2024-10-09T08:24:12.9534447Z Download action repository 'juliangruber/sleep-action@v2.0.0' (SHA:0ec9cf4f8d053aae56e4b402438184a31c6e87d3)
2024-10-09T08:24:13.7652072Z Download action repository 'actions/download-artifact@v4' (SHA:fa0a91b85d4f404e444e00e005971372dc801d16)
2024-10-09T08:24:14.4796415Z Complete job name: deploy-job
2024-10-09T08:24:15.2206463Z ##[group]Checking docker version
2024-10-09T08:24:15.2232516Z ##[command]/usr/bin/docker version --format '{{.Server.APIVersion}}'
2024-10-09T08:24:15.2652155Z '1.44'
2024-10-09T08:24:15.2705001Z Docker daemon API version: '1.44'
2024-10-09T08:24:15.2706190Z ##[command]/usr/bin/docker version --format '{{.Client.APIVersion}}'
2024-10-09T08:24:15.3087314Z '1.44'
2024-10-09T08:24:15.3118463Z Docker client API version: '1.44'
2024-10-09T08:24:15.3127776Z ##[endgroup]
2024-10-09T08:24:15.3133418Z ##[group]Clean up resources from previous jobs
2024-10-09T08:24:15.3142706Z ##[command]/usr/bin/docker ps --all --quiet --no-trunc --filter "label=7e733d"
2024-10-09T08:24:15.3494527Z ##[command]/usr/bin/docker network prune --force --filter "label=7e733d"
2024-10-09T08:24:15.3833431Z ##[endgroup]
2024-10-09T08:24:15.3834068Z ##[group]Create local container network
2024-10-09T08:24:15.3851474Z ##[command]/usr/bin/docker network create --label 7e733d github_network_136f002a4b5b4ee7924d9dc86eb21a60
2024-10-09T08:24:15.4947517Z 5d9a413c4fca510ed299c71a546c53dfb7549dd9f1e1096c4970b7e975ec0456
2024-10-09T08:24:15.4983949Z ##[endgroup]
2024-10-09T08:24:15.5115914Z ##[group]Starting job container
2024-10-09T08:24:15.5211051Z ##[command]/usr/bin/docker --config /home/bmeuer/actions-runner/_work/_temp/.docker_880ffd96-c45b-4475-ba51-d5907752b200 login ghcr.io -u bjmeuer --password-stdin
2024-10-09T08:24:15.9963908Z ##[command]/usr/bin/docker --config /home/bmeuer/actions-runner/_work/_temp/.docker_880ffd96-c45b-4475-ba51-d5907752b200 pull ghcr.io/aristanetworks/avd/universal:latest
2024-10-09T08:24:16.5519020Z latest: Pulling from aristanetworks/avd/universal
2024-10-09T08:24:16.6417996Z Digest: sha256:98ec3957ef1b7d4bc29b989636d4d350e53a4bfff38d67f5eb82771723c5636b
2024-10-09T08:24:16.6420133Z Status: Image is up to date for ghcr.io/aristanetworks/avd/universal:latest
2024-10-09T08:24:16.6441988Z ghcr.io/aristanetworks/avd/universal:latest
2024-10-09T08:24:16.6732536Z ##[command]/usr/bin/docker create --name 71d8db1b55aa4c1ab2daf31cb03f146f_ghcrioaristanetworksavduniversallatest_99ea30 --label 7e733d --workdir /__w/L3LS_Automation/L3LS_Automation --network github_network_136f002a4b5b4ee7924d9dc86eb21a60  -e "PYTHONPATH=/home/avd/.local/lib/python3.11/site-packages" -e "HOME=/github/home" -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/bmeuer/actions-runner/_work":"/__w" -v "/home/bmeuer/actions-runner/externals":"/__e":ro -v "/home/bmeuer/actions-runner/_work/_temp":"/__w/_temp" -v "/home/bmeuer/actions-runner/_work/_actions":"/__w/_actions" -v "/home/bmeuer/actions-runner/_work/_tool":"/__w/_tool" -v "/home/bmeuer/actions-runner/_work/_temp/_github_home":"/github/home" -v "/home/bmeuer/actions-runner/_work/_temp/_github_workflow":"/github/workflow" --entrypoint "tail" ghcr.io/aristanetworks/avd/universal:latest "-f" "/dev/null"
2024-10-09T08:24:16.7529519Z ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:24:16.7584644Z ##[command]/usr/bin/docker start ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:24:17.3804999Z ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:24:17.3841931Z ##[command]/usr/bin/docker ps --all --filter id=ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb --filter status=running --no-trunc --format "{{.ID}} {{.Status}}"
2024-10-09T08:24:17.4111542Z ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb Up Less than a second
2024-10-09T08:24:17.4159878Z ##[command]/usr/bin/docker inspect --format "{{range .Config.Env}}{{println .}}{{end}}" ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:24:17.4447656Z PYTHONPATH=/home/avd/.local/lib/python3.11/site-packages
2024-10-09T08:24:17.4448649Z HOME=/github/home
2024-10-09T08:24:17.4449377Z GITHUB_ACTIONS=true
2024-10-09T08:24:17.4451768Z CI=true
2024-10-09T08:24:17.4453581Z PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/avd/.local/bin
2024-10-09T08:24:17.4454898Z LANG=C.UTF-8
2024-10-09T08:24:17.4455611Z GPG_KEY=A035C8C19219BA821ECEA86B64E628F8D684696D
2024-10-09T08:24:17.4456415Z PYTHON_VERSION=3.11.9
2024-10-09T08:24:17.4457114Z PYTHON_PIP_VERSION=24.0
2024-10-09T08:24:17.4457823Z PYTHON_SETUPTOOLS_VERSION=65.5.1
2024-10-09T08:24:17.4459161Z PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/dbf0c85f76fb6e1ab42aa672ffca6f0a675d9ee4/public/get-pip.py
2024-10-09T08:24:17.4460956Z PYTHON_GET_PIP_SHA256=dfe9fd5c28dc98b5ac17979a953ea550cec37ae1b47a5116007395bfacff2ab9
2024-10-09T08:24:17.4462139Z ANSIBLE_CONFIG=ansible.cfg
2024-10-09T08:24:17.4462839Z DOCKER_BUILDKIT=1
2024-10-09T08:24:17.4510654Z ##[endgroup]
2024-10-09T08:24:17.4561494Z ##[group]Waiting for all services to be ready
2024-10-09T08:24:17.4565701Z ##[endgroup]
2024-10-09T08:24:17.5134088Z ##[group]Run juliangruber/sleep-action@v2.0.0
2024-10-09T08:24:17.5135323Z with:
2024-10-09T08:24:17.5136384Z   time: 240s
2024-10-09T08:24:17.5137485Z ##[endgroup]
2024-10-09T08:24:17.5337434Z ##[command]/usr/bin/docker exec  ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb sh -c "cat /etc/*release | grep ^ID"
2024-10-09T08:28:18.1128953Z ##[group]Run sudo chown -R avd:avd $GITHUB_WORKSPACE
2024-10-09T08:28:18.1130542Z [36;1msudo chown -R avd:avd $GITHUB_WORKSPACE[0m
2024-10-09T08:28:18.1136052Z shell: sh -e {0}
2024-10-09T08:28:18.1136757Z ##[endgroup]
2024-10-09T08:28:18.5422939Z ##[group]Run chmod -R 755 $GITHUB_WORKSPACE
2024-10-09T08:28:18.5423698Z [36;1mchmod -R 755 $GITHUB_WORKSPACE[0m
2024-10-09T08:28:18.5424358Z shell: sh -e {0}
2024-10-09T08:28:18.5424759Z ##[endgroup]
2024-10-09T08:28:18.7041138Z ##[group]Run sudo chown -R avd:avd /github
2024-10-09T08:28:18.7041868Z [36;1msudo chown -R avd:avd /github[0m
2024-10-09T08:28:18.7042503Z shell: sh -e {0}
2024-10-09T08:28:18.7042877Z ##[endgroup]
2024-10-09T08:28:18.8589368Z ##[group]Run actions/download-artifact@v4
2024-10-09T08:28:18.8590493Z with:
2024-10-09T08:28:18.8590921Z   name: l3ls_automation
2024-10-09T08:28:18.8591455Z   path: /__w/L3LS_Automation/L3LS_Automation
2024-10-09T08:28:18.8592056Z   merge-multiple: false
2024-10-09T08:28:18.8592544Z   repository: bjmeuer/L3LS_Automation
2024-10-09T08:28:18.8593073Z   run-id: 11250881751
2024-10-09T08:28:18.8593473Z ##[endgroup]
2024-10-09T08:28:18.8598589Z ##[command]/usr/bin/docker exec  ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb sh -c "cat /etc/*release | grep ^ID"
2024-10-09T08:28:19.5518982Z Downloading single artifact
2024-10-09T08:28:19.9064588Z Preparing to download the following artifacts:
2024-10-09T08:28:19.9066277Z - l3ls_automation (ID: 2033631311, Size: 239526)
2024-10-09T08:28:20.1314570Z Redirecting to blob download url: https://productionresultssa18.blob.core.windows.net/actions-results/bf4b7b43-7999-42b3-b44d-b5b0b37e62b9/workflow-job-run-8af296b7-21d4-5c91-6ce3-d0d596a2f63e/artifacts/2468244a785eb4f41926a5a29cfaaa7fb0ec4848c7e2931cbca666ebc6ed2ab9.zip
2024-10-09T08:28:20.1318249Z Starting download of artifact to: /__w/L3LS_Automation/L3LS_Automation
2024-10-09T08:28:20.5178551Z (node:58) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
2024-10-09T08:28:20.5180958Z (Use `node --trace-deprecation ...` to show where the warning was created)
2024-10-09T08:28:20.8815000Z Artifact download completed successfully.
2024-10-09T08:28:20.8815723Z Total of 1 artifact(s) downloaded
2024-10-09T08:28:20.8830946Z Download artifact has finished successfully
2024-10-09T08:28:20.9100688Z ##[group]Run ansible-playbook -i ./l3ls_automation/inventory_test.yml playbooks/fabric-deploy_test.yml
2024-10-09T08:28:20.9102705Z [36;1mansible-playbook -i ./l3ls_automation/inventory_test.yml playbooks/fabric-deploy_test.yml[0m
2024-10-09T08:28:20.9103982Z shell: sh -e {0}
2024-10-09T08:28:20.9104400Z ##[endgroup]
2024-10-09T08:28:24.1200475Z 
2024-10-09T08:28:24.1202182Z PLAY [Configuration upload for Fabric] *****************************************
2024-10-09T08:28:24.1203139Z 
2024-10-09T08:28:24.1203761Z TASK [arista.avd.eos_config_deploy_cvp : Create required output directories if not present] ***
2024-10-09T08:28:25.2347234Z changed: [cvaas -> localhost] => (item=/__w/L3LS_Automation/L3LS_Automation/l3ls_automation/intended/structured_configs/cvp)
2024-10-09T08:28:25.2348307Z 
2024-10-09T08:28:25.2348808Z TASK [arista.avd.eos_config_deploy_cvp : Verify Requirements] ******************
2024-10-09T08:28:25.4505919Z AVD version 5.0.0-dev9
2024-10-09T08:28:25.4506645Z Use -v for details.
2024-10-09T08:28:25.4507908Z ok: [cvaas -> localhost]
2024-10-09T08:28:25.4508255Z 
2024-10-09T08:28:25.4508848Z TASK [arista.avd.eos_config_deploy_cvp : Start creation/update process.] *******
2024-10-09T08:28:25.4958363Z included: /home/avd/.ansible/collections/ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/v3/main.yml for cvaas
2024-10-09T08:28:25.4960402Z 
2024-10-09T08:28:25.4960948Z TASK [arista.avd.eos_config_deploy_cvp : Generate intended variables] **********
2024-10-09T08:28:28.7705240Z ok: [cvaas]
2024-10-09T08:28:28.7705570Z 
2024-10-09T08:28:28.7706245Z TASK [arista.avd.eos_config_deploy_cvp : Build DEVICES and CONTAINER definition for cvaas] ***
2024-10-09T08:28:30.6771458Z changed: [cvaas -> localhost]
2024-10-09T08:28:30.6771920Z 
2024-10-09T08:28:30.6772414Z TASK [arista.avd.eos_config_deploy_cvp : Start creation/update process.] *******
2024-10-09T08:28:30.7709269Z included: /home/avd/.ansible/collections/ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/v3/present.yml for cvaas
2024-10-09T08:28:30.7710486Z 
2024-10-09T08:28:30.7713748Z TASK [arista.avd.eos_config_deploy_cvp : Load CVP device information for cvaas] ***
2024-10-09T08:28:31.0289493Z ok: [cvaas]
2024-10-09T08:28:31.0289802Z 
2024-10-09T08:28:31.0290331Z TASK [arista.avd.eos_config_deploy_cvp : Create configlets on CVP cvaas.] ******
2024-10-09T08:28:33.9681969Z fatal: [cvaas]: FAILED! => {"changed": false, "msg": "JSONSCHEMA is required. Please install using pip install jsonschema"}
2024-10-09T08:28:33.9683017Z 
2024-10-09T08:28:33.9683341Z PLAY RECAP *********************************************************************
2024-10-09T08:28:33.9684449Z cvaas                      : ok=7    changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2024-10-09T08:28:33.9685229Z 
2024-10-09T08:28:34.0995548Z ##[error]Process completed with exit code 2.
2024-10-09T08:28:34.1102421Z Stop and remove container: 71d8db1b55aa4c1ab2daf31cb03f146f_ghcrioaristanetworksavduniversallatest_99ea30
2024-10-09T08:28:34.1111045Z ##[command]/usr/bin/docker rm --force ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:28:34.4685295Z ed47ddf77216f25811f2b28a3e22667fe5ca96cbb9731f8a5a790ce4c7027bbb
2024-10-09T08:28:34.4739244Z Remove container network: github_network_136f002a4b5b4ee7924d9dc86eb21a60
2024-10-09T08:28:34.4748979Z ##[command]/usr/bin/docker network rm github_network_136f002a4b5b4ee7924d9dc86eb21a60
2024-10-09T08:28:34.6352271Z github_network_136f002a4b5b4ee7924d9dc86eb21a60
2024-10-09T08:28:34.6717627Z Cleaning up orphan processes
```


### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

--- a/ansible_collections/arista/avd/docs/release-notes/5.x.x.md
+++ b/ansible_collections/arista/avd/docs/release-notes/5.x.x.md
@@ -17,7 +17,6 @@ title: Release Notes for AVD 5.x.x
 ### Changes to requirements
 
 - AVD v.5.0.0 requires Python version 3.10 or newer.
-- AVD no longer depends on the `jsonschema` Python library.
 - AVD now requires the `cvprac` Python library to be version 1.4.0 or higher.<br>
   The latest version can be installed with `pip install "cvprac>=1.4.0" --upgrade`.
 

--- a/python-avd/pyproject.toml
+++ b/python-avd/pyproject.toml
@@ -41,6 +41,8 @@ ansible-collection = [
     "netaddr>=0.7.19",
     "PyYAML>=6.0.0",
     "treelib>=1.5.5",
+    # jsonschema is required for the arista.cvp Ansible collection
+    "jsonschema>=3.2.0",
 ]
 
 [build-system]

